### PR TITLE
Style/new app styling 228

### PR DESCRIPTION
**Pull request by annapakula**
New styles

image
image

### CODE REVIEW

File: onboarding/src/static/css/WelcomePage.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import './utilities/mixins.scss';
+
+.WelcomePage {
+  width: 100vw;
+  height: 100vh;
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: center no-repeat url("../images/background.jpg") fixed;
```
Comment (adam-sas-on) [inline onboarding/src/static/css/WelcomePage.scss:10]: https://github.com/binawork/OnlineOnboarding/pull/243/files#r633096226

File: onboarding/src/static/css/LeftMenu.scss [hunk 2]
```diff
@@ -34,18 +61,19 @@
   &__pseudo-wrapper {
     width: 100%;
     z-index: 1;
-    background: var(--light);
+    background: center bottom url('../images/background.jpg');
```
Comment (adam-sas-on) [inline onboarding/src/static/css/LeftMenu.scss:64]: https://github.com/binawork/OnlineOnboarding/pull/243/files#r633096226

Comment (annapakula) [reply onboarding/src/static/css/LeftMenu.scss]: done

File: onboarding/src/static/css/WelcomePage.scss (hunk 1 above)
Comment (annapakula) [reply onboarding/src/static/css/WelcomePage.scss]: done

File: onboarding/src/static/css/style.scss [hunk 3]
```diff
@@ -10,10 +11,25 @@
     src: url("../fonts/summernote.woff2") format("woff2");
 }
 
+body {
+    margin: 0;
+    background: center no-repeat url("../images/background.jpg") fixed;
```
[the remark this answers is not in the document]
Comment (annapakula) [reply onboarding/src/static/css/style.scss]: done

**Pull request merged**